### PR TITLE
Changes some of the accent words to be less confusing

### PR DESCRIPTION
**Pull request by ChuChu42**
## About The Pull Request

a doctor and a healer isnt the same thing, while doctors are healers, they arent always one
Support turning into back makes no fucking sense whatsomever

comment under for suggestions

## Why It's Good For The Game

doctor and healer isnt the same thing, one does surgery and gives medicine, one just clicks you once to heal you, doctor shouldnt translate to healer automatically

support being "back" makes no sense and causes sentence to fuckup and be hard to understand

### CODE REVIEW

**Comment by Nakalon** [comment]
> Nword is ebon because ebon

I don't think this is a good idea.
I think it makes it so typing the nword is acceptable in RP.
Any time someone says gnerd you know something funky is about...

**Comment by Nakalon** [comment]
> No mention of the jew to merchant change (jew to gizan should probably just be removed)

Since Jew isn't a slur it's more of a out of universe thing I have no specific feelings towards it.

But I believe no one with good intentions would say it here either...

**Comment by Gyrandola** [comment]
image
This is unironically even more racist lol.

image
Doctor is perfectly fine.

**Comment by Githubaccount222** [comment]
> > No mention of the jew to merchant change (jew to gizan should probably just be removed)
> 
> Since Jew isn't a slur it's more of a out of universe thing I have no specific feelings towards it.
> 
> But I believe no one with good intentions would say it here either...

After commenting, I realized the intent was to make it a more neutral word than "heretic" and felt a little silly. My mistake there.
My thought process was also "no one with good intentions would say it here" too.

**Comment by K4rlox** [comment]
> Back refers to the rear, where support typically is. Healer is just a general term for a physician.

healer and physician isnt the same thing at all and creates confusion

**Comment by K4rlox** [comment]
> image This is ironically even more racist lol.
> 
> image Doctor is perfectly fine.

ebon is in universe, gnerd is not

doctor is someone who heals using surgery and medicine, healer is someone who uses magic, when someone says "im a doctor" they say "im a healer" instead, which creates confusion

**Comment by K4rlox** [comment]
> > No mention of the jew to merchant change (jew to gizan should probably just be removed)
> 
> Since Jew isn't a slur it's more of a out of universe thing I have no specific feelings towards it.
> 
> But I believe no one with good intentions would say it here either...

its was a suggestion by @wow50000

**Comment by Gyrandola** [comment]
> doctor is someone who heals using surgery and medicine, healer is someone who uses magic, when someone says "im a doctor" they say "im a healer" instead, which creates confusion

Uh, yeah? So why are you arguing against your own changes.

**Comment by K4rlox** [comment]
> > doctor is someone who heals using surgery and medicine, healer is someone who uses magic, when someone says "im a doctor" they say "im a healer" instead, which creates confusion
> 
> Uh, yeah? So why are you arguing against your own changes.
wdym
image

File: strings/Anglish.json [hunk 1]
```diff
@@ -88,7 +87,6 @@
         "helmet": "helm",
         "in front": "to the fore",
         "apart": "asunder",
-        "support": "back",
```
Comment (Nakalon) [inline strings/Anglish.json:91]: change support to "aid" if you want.

File: strings/accent_universal.json [hunk 2]
```diff
@@ -34,15 +34,15 @@
 		"jehovah": "zizo",
 		"retarded": "foolish",
 		"retard": "nimrod",
-		"negro": "gnerd",
```
Comment (Nakalon) [inline strings/accent_universal.json:37]: Keep these. Changing them to Ebon is even more racist and will lead to people purposefully typing out these slurs.

**Review by Nakalon — CHANGES_REQUESTED** [review]

**Comment by Nakalon** [comment]
Awesome!

**Review by Nakalon — APPROVED** [review]

**Pull request merged**